src/Core/Grid/Definition/Factory/WebserviceKeyDefinitionFactory.php

```diff
@@ -205,10 +207,11 @@ protected function getFilters()
             ->add(
                 (new Filter('actions', SearchAndResetType::class))
                     ->setTypeOptions([
-                        'attr' => [
-                            'data-url' => $this->resetActionUrl,
-                            'data-redirect' => $this->redirectionUrl,
+                        'reset_route' => 'admin_common_reset_search_by_filter_id',
```

Comment:
Same as `LogGridDefinitionFactory` construction could be cleaned

Reply:
Fixed